authors/list_latex.py

```diff
@@ -7,7 +7,11 @@
 author_list = load(open("authors.json"))
 
 with open("../authors.tex", "wb") as f:
```

Comment:
Actually, this is to make it work in Python 3. It must have been broken by my recent fix, as the file in master works in both Python 2 and Python 3, I tested it. The file in this PR is only tested with Python 2, as I don't have Python 3 here.

Would you mind fixing it for Python 3, and also making sure it works in Python 2?